app/controllers/v0/in_progress_forms_controller.rb

```diff
@@ -26,7 +26,7 @@ def update
       form.update!(form_data: params[:form_data] || params[:formData], metadata: params[:metadata])
 
       if Flipper.enabled?(:intent_to_file_lighthouse_enabled) && form.id_previously_changed? &&
-         Lighthouse::CreateIntentToFileJob::ITF_FORMS.include?(form.form_id)
+         Lighthouse::CreateIntentToFileJob::ITF_FORMS.include?(form.form_id) && @current_user.participant_id.present?
```

Comment:
Should this be?
@current_user?.participant_id?.present?

My rails syntax is bad, but this is more JS style.

Reply:
u mean adding safe nav like this?: `@current_user&.participant_id&.present?`. That would protect against a nil @current_user which is not possible at this point when creating a new in progress form.

Reply:
Sounds good.  Thanks!